iiasa/message_ix
# Change 100 to annual light demand in demand calculation

Removed the dummy number 100 from the demand calculation and replaced it by the annual per capita demand calculated in the line above.  

## How to review
- Sanity check if this was intended for the tutorial or if the magic number 100 was necessary for certain calibrations. 
- Run all of the westeros tutorials and check if the effects described in the tutorials still persist (as now the demand is about 50% smaller).
- update tests to new OBJ.

## PR checklist
